libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/camera/CameraAnimationDelegate.java

```diff
@@ -0,0 +1,37 @@
+package com.mapbox.services.android.navigation.ui.v5.camera;
+
+import com.mapbox.mapboxsdk.camera.CameraUpdate;
+import com.mapbox.mapboxsdk.location.modes.CameraMode;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+
+class CameraAnimationDelegate {
+
+  private final MapboxMap mapboxMap;
+
+  CameraAnimationDelegate(MapboxMap mapboxMap) {
+    this.mapboxMap = mapboxMap;
+  }
+
+  void render(NavigationCameraUpdate update, int durationMs, MapboxMap.CancelableCallback callback) {
+    handle(update, durationMs, callback);
+  }
+
+  private void handle(NavigationCameraUpdate update, int durationMs, MapboxMap.CancelableCallback callback) {
```

Comment:
NIT: this method is not called from anywhere else, it can just be cut and pasted into `render`.